src/components/policy/policy_external/test/policy_manager_impl_test_base.cc

```diff
@@ -83,12 +84,14 @@ struct StringsForUpdate CreateNewRandomData(StringsForUpdate& str) {
   return str;
 }
 
+// cppcheck-suppress unusedFunction
 void CheckIsParamInList(const ::policy::RPCParams& list,
                         const std::string& parameter) {
   EXPECT_TRUE(std::find(list.begin(), list.end(), parameter) != list.end())
       << "Parameter not exists: " << parameter;
 }
 
+// cppcheck-suppress unusedFunction
```

Comment:
@Ypostolov Please, add the comment that this function is used not in this file but in the `policy_manager_impl_test.cc` to avoid some confusing about `cppcheck-suppress unusedFunction`